### PR TITLE
Disable temperature and low voltage overlay warnings only

### PR DESCRIPTION
**Pull request by radujipa**
This change comes amid manufacturing IC specifics for the CKC
battery. As such, we disable the Raspbian overlay warning icons
only and keep the safety mechanism still active (avoid_warnings=2
disables safety as well). Throttling still happens on this setting
and is easily testable by using a programmable power supply or
setting the CPU temp_limit=45 option.

Together with: https://github.com/KanoComputing/kano-updater/pull/252

### CODE REVIEW

**Comment by radujipa** [comment]
@Ealdwulf I've added the `PostUpdate` scenario as well in the Updater to apply the patch to existing users. We need this as we've already shipped a bunch of CKCs.

**Comment by radujipa** [comment]
[(3) As a CK2 Pro user, I don't want to be warned about low voltage levels all of the time [5]](https://trello.com/c/0SUXjR9C/1214-3-as-a-ck2-pro-user-i-dont-want-to-be-warned-about-low-voltage-levels-all-of-the-time-5)

File: boot_default/config.txt [hunk 1]
```diff
@@ -103,4 +103,7 @@ max_usb_current=1
 [pi3]
 # for light board
 enable_uart=1
+
+# Disable temperature and low voltage overlay warnings only, safely is still ON.
```
Comment (tombettany) [inline boot_default/config.txt]: `safely` again :p

Comment (radujipa) [reply boot_default/config.txt]: Suffering from _Shooting From The Hip Syndrome_ ...

**Pull request merged**